cva6/tests/custom/hello_word/hello_word.c

```diff
@@ -0,0 +1,14 @@
+#include <stdint.h>
```

Comment:
Please add a license header.

Reply:
Thanks for the comment @MikeOpenHWGroup, André or me will add header.